pkg/apis/serving/v1/configuration_lifecycle.go

```diff
@@ -24,6 +24,11 @@ import (
 
 var configCondSet = apis.NewLivingConditionSet()
 
+// GetConditionSet retrieves the condition set for this resource. Implements the KRShaped interface.
+func (*Configuration) GetConditionSet() apis.ConditionSet {
+	return configCondSet
```

Comment:
How is this used?

Reply:
This is used by the common code: https://github.com/knative/pkg/blob/master/reconciler/reconcile_common.go

This will allow automatic processing of ObservedGeneration (and hopefully more common logic in the future).